scripts/o3de/o3de/manifest.py

```diff
@@ -501,7 +501,10 @@ def get_project_json_data(project_name: str = None,
     if project_name and not project_path:
         project_path = get_registered(project_name=project_name)
 
-    return get_json_data('project', project_path, validation.valid_o3de_project_json)
+    if pathlib.Path(project_path).is_file():
```

Comment:
This is another place a comment would be useful to know we need this to accommodate project `.json` files that are not named `project.json`